### DISCUSSION TITLE
GitHub achievements

### DISCUSSION THREAD

**Discussion by hzswdef** [community/community#18665] [Other Feature Feedback, Questions, & Ideas] [answered]
how to get "YOLO" achievement?

**Comment by lethetai2605** [comment, marked as answer]
Merged pull request without code review
https://github.com/Schweinepriester/github-profile-achievements

**Comment by Chathu12345** [comment]
> how to get "YOLO" achievement?

To get YOLO achievement badge you have to merge pull requests without code reviewing.
